src/domain/enrolment/CreateEnrolmentPage.tsx

```diff
@@ -28,6 +32,21 @@ const CreateEnrolmentPage: React.FC = () => {
   };
 
   const event = eventData?.event;
+  const eventLocationId = event?.location?.id || '';
```

Comment:
Maybe this could come from the getEventFields now that it is merged to develop branch